app/views/conferences_primer_theme/pages/homepage.html.haml

```diff
@@ -27,5 +27,8 @@
           - if current_conference.present? && current_conference.finish_date >= Date.today
             = link_to t('.more_info'), frontend_conference_path(current_conference),
                       class: %w[btn-mktg btn-large-mktg btn-primary-mktg f3 mt-4]
+          - elsif ::Spina::Page.where(view_template: 'about').any?
+            = link_to t('.more_info'), ::Spina::Page.where(view_template: 'about').first.materialized_path,
```

Comment:
Two suggestions: views aren't evaluated within the context of a particular module, so it will always be the case that `::MyModule == MyModule` in this instance. Also `:name` seems to end up being a unique identifier since it's only set for custom pages, and custom pages can't be created by the user.
```suggestion
            = link_to t('.more_info'), Spina::Page.find_by(name: 'about').materialized_path,
```

Reply:
I had always used global scope just to be safe so that's good to know. Didn't realise the custom page details so this is much better than I thought.